scripts/enumerate_load_table_lookup.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
What do you think about keeping all/most of the code in a module that can be imported by the notebook?